packages/react-scripts/fixtures/kitchensink/src/App.js

```diff
@@ -1,24 +1,33 @@
-import React from 'react';
+import React, { Component, PropTypes, createElement } from 'react';
 
 class BuiltEmitter extends React.Component {
-  constructor(props) {
-    super(props)
-
-    this.callWhenDone = done => done();
+  static propTypes = {
+    feature: PropTypes.func.isRequired
   }
 
   componentDidMount() {
-    this.callWhenDone(() => document.dispatchEvent(new Event('ReactFeatureDidMount')));
+    const { feature } = this.props
+    if (!Component.isPrototypeOf(feature)) {
+      this.notifyRendered();
+    }
   }
 
-  render() {
-    const feature = React.cloneElement(React.Children.only(this.props.children), {
-      setCallWhenDone: done => {
-        this.callWhenDone = done;
-      }
-    });
+  notifyRendered() {
```

Comment:
Naming nit: let's call this method `handleReady`, and call the prop `onReady` everywhere.